examples/helloworld/deployment.yaml

```diff
@@ -11,6 +11,8 @@ spec:
     metadata:
       labels:
         app: helloworld
+        app_label_unique_hash: "{{//helloworld:image__HASH}}"
```

Comment:
How about `.digest` and `.short-digest` suffixes?

Reply:
Sounds goods. I will use digest instead of hash.